Source/ConvolutionFunctions/arm_depthwise_conv_get_buffer_sizes_s8.c

```diff
@@ -32,15 +32,15 @@
 #include "arm_nnsupportfunctions.h"
 
 /**
- *  @ingroup Public
+ *  @ingroup NNConv
  */
 
 /**
- * @addtogroup NNConv
+ * @addtogroup InternalNNConv
  * @{
  */
 
-__STATIC_INLINE int32_t arm_depthwise_conv_s8_opt_get_buffer_size_mve(const cmsis_nn_dims *input_dims,
+static inline int32_t arm_depthwise_conv_s8_opt_get_buffer_size_mve(const cmsis_nn_dims *input_dims,
```

Comment:
what is the reason for this change?

Reply:
Sorry that was a mistake. Leftover from testing.  Done